docs/api/api_docs.yaml

```
@@ -982,6 +982,10 @@ paths:
         '200':
           description: Found attachment successfully.
           headers:
+            Content-Length:
+              schema:
+                type: number
+                description: Length of the attachment in bytes
```

whoops, my suggestion was incorrectly indented

```suggestion
              description: Length of the attachment in bytes
```